src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1496,6 +1496,17 @@
 
   {::vacuum-analyze #{"factsets"}})
 
+(defn resource-events-pk []
+  (jdbc/do-commands
+   "alter table resource_events add column event_hash bytea"
```

Comment:
In the past, I've heard us mention that sometimes altering a table is less efficient that recreating a whole new table. Given that resource_events is probably the largest table in the database do we have any of those concerns here?